apps/docs/app/self-hosting/configuration/page.mdx

```diff
@@ -177,9 +176,8 @@ An example configuration for a FusionAuth OpenID Connect in Formbricks would loo
   <CodeGroup title="Formbricks Env for FusionAuth OIDC">
     ```yml {{ title: ".env" }}
     OIDC_CLIENT_ID=59cada54-56d4-4aa8-a5e7-5823bbe0e5b7
-    OIDC_CLIENT_SECRET=4f4dwP0ZoOAqMW8fM9290A7uIS3E8Xg29xe1umhlB_s
-    OIDC_ISSUER=http://localhost:9011
-    OIDC_DISPLAY_NAME=FusionAuth OIDC_SIGNING_ALGORITHM=HS256
+    OIDC_CLIENT_SECRET=4f4dwP0ZoOAqMW8fM9290A7uIS3E8Xg29xe1umhlB_s OIDC_ISSUER=http://localhost:9011
```

Comment:
looks like there is a small indention issue here :-) (might be related to pretty that is struggling with this file)